doc/api/cli.md

```diff
@@ -626,6 +626,26 @@ Make built-in language features like `eval` and `new Function` that generate
 code from strings throw an exception instead. This does not affect the Node.js
 `node:vm` module.
 
+### `--expose-gc`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental. This flag is inherited from V8 and is subject to
+> change upstream.
+
+This flag will expose the gc extension from V8.
+
+```js
+// If the --expose-gc flag is provided, V8 will expose the garbage collector
+// allowing users to trigger garbage collection on demand.
```

Comment:
I'd prefer that we don't document this because what "trigger garbage collection" means is usually not what users expected (the kind of garbage collection that `gc()` does isn't very eager, it can leave unreachable objects around, which isn't a bug because it's only invented for V8's testing needs and only cater to what their tests need, but that can be very surprising to users expecting or even relying on it to "clean up all the garbage in the heap if they are already unreachable"). We already saw that happenning to Jest's leak detector (even after doing `gc()` 10 times it can still produce false positives) https://github.com/nodejs/node/pull/48510#issuecomment-1719038812

Reply:
I can remove the docs, or specify that is for testing purposes only and it is not meant to be ran in productive environments. Or do u have any suggestion for this?

Reply:
+1 to *not* documenting as Joyee suggests.